### ISSUE TITLE
Proper display of TEI header in GUI?

### ISSUE THREAD

**Issue by rsimon** [hdcaicyt/Recogito-TEI#19]
A question just brough up by @eltonteb: should Recogito display the TEI header information in the annotation view? Or should everything inside `teiHeader` remain hidden? (Or even be made collapsible etc.?)

**Comment by eltonteb** [comment]
Can such info even be automatically ingested into Recogito's metadata view? I mean, this is all useful info which I don't want it in the annotation UI, but which I'm currently having to type out myself in the metadata screen.

**Comment by rsimon** [comment]
Yes, that's also discussed in the previous issue ;-) https://github.com/hdcaicyt/Recogito-TEI/issues/18

**Comment by eltonteb** [comment]
lucky someone is keeping track of all these issues...

**Comment by Gimena** [comment]
But don't we have two different scenarios/also two different knid of users here? We may have one case in which someone is uploading a txt file and know nothing about TEI, but what would bring together both cases and help having a more unified recogito would be to have  the same fields in the metadata in the TEI file and viceversa. That's why @hdcaicyt Nidia and I started a beta file revising those fields/TEI tags, and also did with Gustavo.

**Comment by GusRiva** [comment]
I agree with Gimena. The best would be to make the recogito metadata fields be TEI compliant.

**Comment by arojascastro** [comment]
@rsimon will provide more documentation about Recogito metadata and we will suggest a mapping with TEI.

**Comment by rsimon** [comment]
Ok, as I said, there is no (and never was supposed to be) actual metadata management functionality in Recogito. What get's recorded is simply a number of fields in the database, which we defined spontaneously, a long time ago. The fields are really there for display purposes, i.e. so that Recogito has something to show in the UI, and so that users have some control over what the header of the annotation page looks like (especially relevant when making documents public.)

The fields are:

-  __owner__ the Recogito user who uploaded the document (filled automatically)
-  __uploaded_at__ the time at whicht the document was uploaded to Recogito (filled automatically)
- __title__ a mandatory display name for use in the UI
- __author__ optional, the author of the source
- __date_freeform__ an optional text (not number/timestamp!) field, e.g. to say something about
  the dating of the source
- __description__ free-form text description, optional
- __language__ just a text field, but should contain a 2- or 3-letter ISO code. The code also goes into
  the page markup; and some language codes will trigger RTL display (ar, he, fa), optional
- __source__  a URL to an online source from where the document is from, optional
- __edition__ freetext field, optional
- __license__ optional, filled through a dropdown with a number of possible values (from "copyrighted" to CC0). Affects whether a document can be shared publicly.
- __attribution__ optional, a free-text field for providing extra attribution, along with the license (supports markdown)
- __public_visibility__ filled from the sharing settings (private, public, etc.)
- __public_access_level__ filled from the sharing settings (access to full content, data only, write access for all, etc.)

FYI: the full SQL schema for this is [here](https://github.com/pelagios/recogito2/blob/master/conf/schema.sql#L60-L77).

**Comment by chiarcos** [comment]
A minimal requirement would be to remove TEI *meta*data (i.e., //teiHeader) from text view. Opening https://titus.fkidg1.uni-frankfurt.de/cordon/data/tei/DPN_SD_1869.tei.xml via https://recogito.pelagios.org/ will display metadata CDATA content as part of the text. People familiar with TEI will understand that easily, lay annotators won't.
